doc/quickstart.md

```diff
@@ -5,14 +5,14 @@
 vfkit is a macOS command-line-based hypervisor, which uses [Apple's Virtualization Framework](https://developer.apple.com/documentation/virtualization?language=objc) to run virtual machines.
 You start a virtual machine by running vfkit with a set of arguments describing the virtual machine configuration/hardware.
 When vfkit stops, the virtual machine stops running.
-It requires macOS 11 or newer, and runs on both x86_64 and aarch64 Macs.
+It requires macOS 11 or newer, and runs on both x86_64 and arm64 Macs.
```

Comment:
I usually use either `amd64`/`arm64` which is the go naming, or `x86_64`/`aarch64` which is the linux naming, `x86_64`+`arm64` is imo weirdly inconsistent. We can use `Intel and Apple silicon Macs` which is the naming used by Apple.